2748. Number of Beautiful Pairs/solution.py

```diff
@@ -0,0 +1,31 @@
+class Solution:
+    # This takes in a list of nums and returns each beautiful pair,
+    # where a beautiful pair is when the first digit of the first
+    # num in the pair is coprime with the last digit of the second
+    # num in the pair. Two nums are coprime if their gcd is no greater
+    # than 1.
+    def countBeautifulPairs(self, nums: List[int]) -> int:
+        # the number of beautiful pairs
+        
+        # convert each int in the list to strings to save
+        # time from converting each int to a string inside the 
+        # main loop
+        
+        # iterate to the second-to-last digit of nums
+        
+            # iterate through the next nums
```

Comment:
probably more clear to say iterate from the next element in nums to the end of nums